docs/api/shallow.md

```diff
@@ -169,9 +169,6 @@ Manually sets context of the root component.
 #### [`.instance() => ReactComponent`](ShallowWrapper/instance.md)
 Returns the instance of the root component.
 
-#### [`.update() => ShallowWrapper`](ShallowWrapper/update.md)
```

Comment:
This method still seems useful in case someone mutates state/props without calling the appropriate methods and wants to force an update.